app/src/main/java/org/kinecosystem/kinit/view/earn/WebTaskTruexFragment.kt

```diff
@@ -16,6 +18,15 @@ import org.kinecosystem.kinit.viewmodel.earn.WebFragmentActions
 import org.kinecosystem.kinit.viewmodel.earn.WebTaskTruexViewModel
 
 class WebTaskTruexFragment : BaseFragment(), WebFragmentActions {
+    override fun openBrowser(url: String) {
+        val parsedUrl = Uri.parse(url)
+        if (parsedUrl == null) {
+            showErrorDialog()
+        } else {
+            startActivity(Intent(Intent.ACTION_VIEW, parsedUrl))
```

Comment:
need to check first there is an app on the device that can view urls (i.e. a browser) 
You do this by adding if as follows:
```
if (intent.resolveActivity(getPackageManager()) != null) {
    startActivity(Intent(Intent.ACTION_VIEW, parsedUrl()) 
}
```
see https://developer.android.com/guide/components/intents-common#ViewUrl